recipes-connectivity/openssh/files/wolfssl-openssh-8.5p1.patch

```diff
@@ -0,0 +1,531 @@
+diff --git a/authfd.c b/authfd.c
+index 9f092f7c..9044c868 100644
+--- a/authfd.c
++++ b/authfd.c
+@@ -505,7 +505,7 @@ ssh_add_identity_constrained(int sock, struct sshkey *key,
+ 		return SSH_ERR_ALLOC_FAIL;
+ 
+ 	switch (key->type) {
+-#ifdef WITH_OPENSSL
++#if defined(WITH_OPENSSL)
```

Comment:
Are these switches from `#ifdef` to `#if defined` necessary?  Also one below this in authfile.c.